pylama.ini

```diff
@@ -40,14 +40,15 @@ ignore = W0703
 # W0703 Catching too general exception Exception [pylint]
 
 [pylama:cloudmarker/test/test_*.py]
-ignore = D102,C0111,R0903,R0904,R0201,W0613
+ignore = D102,C0111,R0903,R0904,R0201,W0613,W0212
 
 # D102 Missing docstring in public method [pydocstyle]
 # C0111 Missing method docstring [pylint]
 # R0903 Too few public methods (0/2) [pylint]
 # R0904 Too many public methods (23/20) [pylint]
 # R0201 Method could be a function [pylint]
 # W0613 Unused argument 'mock_service_account' in test_gcpcloud.py
+# W0212 Access to a protected member _buffer of a client class
```

Comment:
Once the tests are updated to not rely on internal members, we can get rid of this change.